src/app/core/models/issue-dispute.model.ts

```diff
@@ -0,0 +1,34 @@
+export class IssueDispute {
+    title: string; // e.g Downgrade of severity
+    description: string; // e.g Team says: xxx\n Tester says: xxx.
+    tutorResponse: string; // e.g Not justified. I've changed it back.
+    todo: string; // e.g  - [x] Done
+
+    constructor(title: string, description: string) {
+        this.title = title;
+        this.description = description;
+        this.tutorResponse = '[replace this with your explanation]';
+        this.todo = '- [ ] Done';
```

Comment:
For `this.todo`, can we make this as a constant like `NOT_DONE` or `UNCHECKED_TODO` or something along those lines? That way at least if we were to make changes to the formatting it will be in one place.

Perhaps the same can be done for the `this.tutorResponse`?